src/code/z_kankyo.c

```diff
@@ -510,7 +525,302 @@ s32 Environment_ZBufValToFixedPoint(s32 zBufferVal) {
     return ret;
 }
 
+#ifdef NON_MATCHING
```

Comment:
I'm kind of wondering if we should mention that this has the exact same problem as `func_809F24C8` in `Boss06`?